<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Adduser</class>
 <widget class="QDialog" name="Adduser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add User</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>591</width>
     <height>621</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="holder">
    <item>
     <layout class="QHBoxLayout" name="innerHolder">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="Form">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="2">
           <widget class="QSpinBox" name="AgeSpin"/>
          </item>
          <item row="6" column="1">
           <widget class="QLabel" name="RoleTag">
            <property name="text">
             <string>Role:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="LastTag">
            <property name="text">
             <string>Last Name:</string>
            </property>
            <property name="buddy">
             <cstring>LastLine</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="FirstLine"/>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="AgeTag">
            <property name="text">
             <string>Age:</string>
            </property>
            <property name="buddy">
             <cstring>AgeSpin</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLineEdit" name="UserLine"/>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="FirstTag">
            <property name="text">
             <string>First Name:</string>
            </property>
            <property name="buddy">
             <cstring>FirstLine</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="LastLine"/>
          </item>
          <item row="6" column="2">
           <layout class="QHBoxLayout" name="RoleField">
            <item>
             <widget class="QRadioButton" name="AdminRadio">
              <property name="text">
               <string>Administrator</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="UserRadio">
              <property name="text">
               <string>User</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="2">
           <widget class="QLineEdit" name="PassLine"/>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="UserTag">
            <property name="text">
             <string>Username</string>
            </property>
            <property name="buddy">
             <cstring>UserLine</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="PassTag">
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="text">
             <string>Password</string>
            </property>
            <property name="buddy">
             <cstring>PassLine</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="SaveBtn">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>FirstLine</tabstop>
  <tabstop>LastLine</tabstop>
  <tabstop>AgeSpin</tabstop>
  <tabstop>AdminRadio</tabstop>
  <tabstop>UserRadio</tabstop>
  <tabstop>SaveBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
